Bottom: 8198535745044021fbdf200bf835967b76f06847
Top:    e67eb4a93b7515014402ae0647d0a76dece141c3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:42:27 +0000

Refresh of improve-dump_poly-output

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ce74155..d9f6ce4 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -159,7 +159,7 @@ poly_dump (POLYAREA * p)
           fprintf (stderr, "NEXT PLINE\n");
         }
       while ((pl = pl->next) != NULL);
-      fprintf (stderr, "NEXT_POLY\n");
+      fprintf (stderr, "NEXT POLY\n");
     }
   while ((p = p->f) != f);
 }
